docs/en/observability/metrics-threshold-alert.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[metrics-threshold-alert]]
+= Create a metrics threshold alert
+
+Based on the metrics that are listed on the *Metrics Explorer* page within the {metrics-app},
+you can create a threshold alert to notify you when a metric has reached or exceeded a value for a specific
+time period.
+
+Additionally, each alert can be defined using multiple
+conditions that combine metrics and thresholds to create precise notifications.
+
+. In the side navigation, expand *Observability*, click *Metrics*, and then click *Metrics Explorer*.
+. On the *Metrics Explorer* page, click *Alerts*, and then select *Create alert*.
+
```

Comment:
Alerts can also be created from every graph in Metrics Explorer. Moreover, choosing this option will pre-fill the condition, filter and create per sections of a threshold alert.
<img width="433" alt="Screen Shot 2020-09-23 at 14 34 28" src="https://user-images.githubusercontent.com/1685068/94013398-4c06c080-fdaa-11ea-9a3b-867e3a75cc86.png">

Reply:
Added this to the docs.